packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -97,9 +97,10 @@ export function FileUploader({
         setFileStatuses((prevFileStatuses) => {
           const prevStatus = { ...prevFileStatuses[index] };
 
-          const progressPercentage = Math.floor(
-            (progress.loaded / progress.total) * 100
-          );
+          const progressPercentage =
+            progress.total !== 0
```

Comment:
Can we add a comment here that this is checking for zero byte size files? In the future, I think this will be confusing to read.

Reply:
Ok added comment above it [46db5b1](https://github.com/aws-amplify/amplify-ui/pull/3260/commits/46db5b11dc514c729ce3e53498f1dad6ecb70712)